cmd2/AutoCompleter.py

```diff
@@ -0,0 +1,796 @@
+# coding=utf-8
+import argparse
+import re as _re
+import sys
+from argparse import OPTIONAL, ZERO_OR_MORE, ONE_OR_MORE, REMAINDER, PARSER, ArgumentError, _
+from .rl_utils import rl_force_redisplay
```

Comment:
In general I try to order imports like so:
1st:  built-in Python modules
2nd: 3rd-party modules
3rd: Our code
(and each section is alphabetical within that).

So recommend moving import from .rl_utils to end.